velox/vector/arrow/Bridge.cpp

```diff
@@ -972,7 +972,17 @@ TypePtr importFromArrowImpl(
         // Parse "d:".
         int precision = std::stoi(&format[2], &sz);
         // Parse ",".
-        int scale = std::stoi(&format[2 + sz + 1], &sz);
+        int idx = 2 + sz + 1;
+        int scale = std::stoi(&format[idx], &sz);
+        // Handle bitwidth.
+        if (format[idx + sz] == ',') {
+          int bitWidth = std::stoi(&format[idx + sz + 1], &sz);
```

Comment:
Would above fail for an invalid format like `d:5,2,`?

Reply:
Yeap, it will be caught by `std::invalid_argument&`

Reply:
Would you add a unit test and assert the exception? Thanks.

Reply:
yeap, already added